.github/actions/deploy-windows/action.yml

```diff
@@ -105,13 +106,17 @@ runs:
           rem cudnn_8.8.0.121_windows.exe -s
           unzip cudnn-windows-x86_64-8.9.5.29_cuda12-archive.zip
           unzip zlib123dllx64.zip
+          unzip nvcomp_3.0.4_windows_12.x.zip
           rem move "%ProgramFiles%\NVIDIA\CUDNN\v8.8\bin\*.dll" "%ProgramFiles%\NVIDIA GPU Computing Toolkit\CUDA\v12.3\bin"
           rem move "%ProgramFiles%\NVIDIA\CUDNN\v8.8\include\*.h" "%ProgramFiles%\NVIDIA GPU Computing Toolkit\CUDA\v12.3\include"
           rem move "%ProgramFiles%\NVIDIA\CUDNN\v8.8\lib\x64\*.lib" "%ProgramFiles%\NVIDIA GPU Computing Toolkit\CUDA\v12.3\lib\x64"
           move cudnn-windows-x86_64-8.9.5.29_cuda12-archive\bin\*.dll "%ProgramFiles%\NVIDIA GPU Computing Toolkit\CUDA\v12.3\bin"
           move cudnn-windows-x86_64-8.9.5.29_cuda12-archive\include\*.h "%ProgramFiles%\NVIDIA GPU Computing Toolkit\CUDA\v12.3\include"
           move cudnn-windows-x86_64-8.9.5.29_cuda12-archive\lib\x64\*.lib "%ProgramFiles%\NVIDIA GPU Computing Toolkit\CUDA\v12.3\lib\x64"
           move dll_x64\zlibwapi.dll "%ProgramFiles%\NVIDIA GPU Computing Toolkit\CUDA\v12.3\bin"
+          xcopy "include" "%ProgramFiles%\NVIDIA GPU Computing Toolkit\CUDA\v12.3\include" /e /s
```

Comment:
We should be able to move everything from there too?

Reply:
I used xcopy for the recursive nature to save some lines

Reply:
I'm pretty sure move the move command can move directories as well, no?

Reply:
It can, but it required extra lines, because we can't "merge" nvcomp `include` dir with CUDA Toolkit `include` dir recursively. If you try to do this, it fails with a "Access is denied".

Reply:
I see, the move command doesn't match directories with wildcards.